HARK/ConsumptionSaving/ConsIndShockModel.py

```diff
@@ -1708,9 +1709,12 @@ def sim_birth(self, which_agents):
             seed=self.RNG.randint(0, 2 ** 31 - 1)
         ).draw(N)
         self.t_age[which_agents] = 0  # How many periods since each agent was born
-        self.t_cycle[
-            which_agents
-        ] = 0  # Which period of the cycle each agent is currently in
+        
```

Comment:
Think about whether you need to add a test for whether the PerfMITShk variable exists, like:

if hasattr(self, 'PerfMITShk'):

before using the variable